basic-features.md

```diff
@@ -4,77 +4,127 @@ summary: Learn about the basic features of TiDB.
+
+| Data types, functions and operators                                                  | 5.0   | 4.0 | 3.1 | 3.0 | 2.1 |
+|--------------------------------------------------------------------------------------|:-----:|:---:|:---:|:---:|:---:|
+| [Numeric types](/data-type-numeric.md)                                               | Y     | Y   | Y   | Y   | Y   |
+| [Date and time types](/data-type-date-and-time.md)                                   | Y     | Y   | Y   | Y   | Y   |
+| [String types](/data-type-string.md)                                                 | Y     | Y   | Y   | Y   | Y   |
+| [JSON type](/data-type-json.md)                                                      | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Control flow functions](/functions-and-operators/control-flow-functions.md)         | Y     | Y   | Y   | Y   | Y   |
+| [String functions](/functions-and-operators/string-functions.md)                     | Y     | Y   | Y   | Y   | Y   |
+| [Numeric functions and operators](/functions-and-operators/numeric-functions-and-operators.md) | Y     | Y   | Y   | Y   | Y   |
+| [Date and time functions](/functions-and-operators/date-and-time-functions.md)       | Y     | Y   | Y   | Y   | Y   |
+| [Bit functions and operators](/functions-and-operators/bit-functions-and-operators.md) | Y     | Y   | Y   | Y   | Y   |
+| [Cast functions and operators](/functions-and-operators/cast-functions-and-operators.md) | Y     | Y   | Y   | Y   | Y   |
+| [Encryption and compression functions](/functions-and-operators/encryption-and-compression-functions.md) | Y     | Y   | Y   | Y   | Y   |
+| [Information functions](/functions-and-operators/information-functions.md)           | Y     | Y   | Y   | Y   | Y   |
+| [JSON functions](/functions-and-operators/json-functions.md)                         | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Aggregation functions](/functions-and-operators/aggregate-group-by-functions.md)    | Y     | Y   | Y   | Y   | Y   |
+| [Window functions](/functions-and-operators/window-functions.md)                     | Y     | Y   | Y   | Y   | Y   |
+| [Miscellaneous functions](/functions-and-operators/miscellaneous-functions.md)       | Y     | Y   | Y   | Y   | Y   |
+| [Operators](/functions-and-operators/operators.md)                                   | Y     | Y   | Y   | Y   | Y   |
+| **Character sets**                                                                   | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [`utf8`](/character-set-and-collation.md)                                            | Y     | Y   | Y   | Y   | Y   |
+| [`utf8mb4`](/character-set-and-collation.md)                                         | Y     | Y   | Y   | Y   | Y   |
+| [`ascii`](/character-set-and-collation.md)[1]                                        | Y     | Y   | Y   | Y   | Y   |
```

Comment:
I have switched it to use footnotes: https://www.markdownguide.org/extended-syntax/#footnotes

This is not supported by GitHub flavored markdown though, so it will make it harder to preview for those that are not running the [web toolchain](https://github.com/pingcap/website-docs) locally. It looks like this:

![image](https://user-images.githubusercontent.com/67764674/94290664-c0ab3d80-ff17-11ea-803c-a863ca0690eb.png)

![image](https://user-images.githubusercontent.com/67764674/94290701-cd2f9600-ff17-11ea-96ef-50b98dee409d.png)

Reply:
@YiniXu9506 Could you help check whether PingCAP web pages support this syntax? Thanks!

Reply:
The footnote reference is correct, but the footnote at the bottom of this page is incorrect. 

`Add the footnote using another caret and number inside brackets with a colon and text ([^1]: My footnote.). `

@nullnotnil You need to add the footnote **_with a colon_**, otherwise,  it won't scroll to the reference correctly.

So here, the footnote should be added like this:

```
[^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details.
[^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.
```

Reply:
Thanks, fixed

Reply:
No problem. LGTM.

FYI, since the footnote is alphabetical order, it will display like this: 
![image](https://user-images.githubusercontent.com/34967660/95819443-98657080-0d58-11eb-8f32-c3854c778d9e.png)